.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: groovy
+after_success:
+- ./gradlew bintrayUpload
+env:
+  global:
+  - secure: lKbFiKVqDOV08Q9WXJXn751r5PejPESxMmvNgLej0cp8tTArzY4JWtbXKuS97/n0KtefHv7QDQanOPt9sXlgm9tVIu6xXvh3hqKDRSkvV8tC8W1I95i8MItFlA+ErrSKCHxL36RHdILPuNiWRh9wxhMFAzsR8LlwU/b0aC6qVd8RjGqrmX3uoh9iXEJcjo87qGO4nBE+oybk+Cp6C+K7WRgRLdFozjj3hLCEQiyUiV8FDH6rRhhQlTa0o8HjTvZdleNXloh2bQjvS1Nj/7nKoplkTn0pyajYh/Qf2KlDTdXW35Wq2UV9XJqnvg/MFia4jxBZIluzQojh0q9QsQEXUyDEFfV5KkiqE5KuFtzldMZcyKSJouBZcwapLaPTDrlWMNrC6djWSYr+H9/92NvORLiMJEJZiVlONOEDQoNE4Uz9L5bzroq5zyOC9A/PWz/8D0dTbkuEaoINYdxdacUt7jDRNRa/MRm3NABe/r+7OsuwLcGry+Nd2BPT4rfG5DH78vdhRDiJBLI7epFFvYwwTKmMY0j6p8sgQ0SJGxLZsck1qXCSHbtzSh0u6YcuXfmXvOXL+LZB8Yx3etM84nBjn3aHfMyf6174K+xdhQtYcmf1P6STpjoMqXKNvg3mvezCSv9d27houLXiiGDcxcly9RnPnYD/wWlPGjDBZYC5bFw=
+  - secure: fzOXDF1coTKocNWAvqDnf3UWbu+OJPn8Z2X7r0qlaruPUteZOLFCfYf45aD8UPxvFp8S6O1EDsbODJXy66kL/Q/fkMyYymoy8lawp6P0r3rWaVffjJmvWcQ9ToB0jGmi674j18H/OrQlt9QB1RhK1zvfbkBYunmaCZASBIxHvvs4nHdaFGoaeaXKNbBWbmxOt9FuqwfLq3Tqn+Xg7R8bt3DJXwBIzsQUn2E6BUdVKC509ZLBQtEJSDI+U/xpTQV5ywBoFTmxylSVlXW0L/QpiOqf4dyp+/jr6s6AMfU3EInMDAmHSjeA7GzhtZMqzSuxIvOkBJPMOJrsaeS+gEm9A15xwLbtVjbLzGKsMrAK9q0XUNt6hHhIYq82rRheN8meuZt6S/kIpV5CllYX8EkS0kQBNlp/6vgo0T7F/ioB65IUD0Q5vr2Dinb/tOPoRbR8uh5itiamZkIU4wFGr69QCwzukubNvmU90fp4u+/lJUYDbE3AfCgL4NfLHh8fC/TamXyBnVKj9HFbI0dtjIxaYxkgb60k2kPafw+w//gAggTkneybxTi79ndyPnLhN9/4+ttsTgi3wWkMxjOAUfpxD4fkx/G65Qfgw6xBTlHri81i4eeCNAvhA97/DijPIiEVRjxNZVVtntZyaMXuoXWerpsh0uXWrhWTLbDzDB4MBKw=
```

Comment:
these are the encrpyted bintray user and API key which will be added as environment variables